packages/app/src/store/wallet/reducer.ts

```diff
@@ -2,9 +2,9 @@ import { Reducer } from 'redux';
 import { WalletActions, WalletState, RESTORE_WALLET, IS_RESTORING_WALLET, GENERATE_WALLET, SIGN_OUT } from './types';
 
 const initialState: WalletState = {
-  seed: null,
+  secretKey: undefined,
```

Comment:
What prompted `undefined` over `null`? I find `null` makes it clearer to the reader it was declared as null explicitly, where as undefined could be indicate it was just never declared at all.

Reply:
I was getting a type error, fixing it required to change to undefined.